tfs/src/main/java/hudson/plugins/tfs/TeamPushTrigger.java

```diff
@@ -48,6 +49,9 @@ public TeamPushTrigger(final Job<?, ?> job) {
         this.job = job;

```

Comment:
In a future PR we should figure out whether this constructor can be removed.

Reply:
Sure, will try that out in the PR to add refspec param into TeamPushTrigger.

Reply:
Okay sounds good.  It might be good for the trigger to inject a parameter for both the refspec and the hash.  That way the same job can be used for both PRs and pushes if desired.

Reply:
Thanks Matt. I tried multiple approaches and chose the environment contributor approach which worked locally. Now both values (vstsRefspec and vstsBranch) are shown in the "Parameters" tab on build page, and it's loadable in triggered builds.

![image](https://user-images.githubusercontent.com/17461693/30766550-aa3f07ae-9faa-11e7-97e2-9fdba00857f5.png)